services/dumper.js

```diff
@@ -163,11 +163,14 @@ function Dumper(config) {
       };
     });
 
+    const modelSafeName = stringUtils.transformToSafeString(table);
+
     copyHandleBarsTemplate({
       source: `app/models/${config.dbDialect === 'mongodb' ? 'mongo' : 'sequelize'}-model.hbs`,
       target: `models/${tableToFilename(table)}.js`,
       context: {
-        modelName: stringUtils.pascalCase(table),
+        modelVariableName: stringUtils.pascalCase(modelSafeName),
+        modelName: stringUtils.camelCase(modelSafeName),
```

Comment:
My last suggestion
```suggestion
        modelName: getModelNameFromTableName(table),
```
Yes it is certainly less performant in terms of code execution but much more maintainable because we understand the the intent using `getModelNameFromTableName` versus the current code.